internal/scorecard/alpha/bundle.go

```diff
@@ -22,18 +22,18 @@ import (
 	"k8s.io/apimachinery/pkg/util/rand"
 )
 
-// getBundleData tars up the contents of a bundle from a path, and returns that tar file in []byte
-func getBundleData(bundlePath string) (bundleData []byte, err error) {
+// GetBundleData tars up the contents of a bundle from a path, and returns that tar file in []byte
+func (o Scorecard) GetBundleData() (bundleData []byte, err error) {
```

Comment:
Nit: Do we plan to use this in `cmd/operator-sdk/alpha/scorecard/cmd.go`? If not any reason it needs to be exported?

Reply:
I un-exported that one.